server/docker/Dockerfile.base-env

```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:22.04
 
+ARG TARGETARCH
```

Comment:
Regarding having a separate arm64 container, I think the biggest benefit is we don't have to fiddle with installing correct cross compile libraries.

As a drawback I wonder what the performance loss is, as I assume this container will still run on a X64 machine.
Regardless, I think this approach is best and easiest, and I don't expect much traffic to this container tbh.

Reply:
Here we build a multiarch image (https://docs.docker.com/build/building/multi-platform/). E.g. that we build 2 images (for linux/amd64 and for linux/arm64) which are merged into single image. Than Docker pull image according to host arch or to `--platform` flag.

Arm64 container will be run at arm64 host, so no performance issue will be.